packages/keyring-controller/src/KeyringController.test.ts

```diff
@@ -595,11 +608,8 @@ describe('KeyringController', () => {
               },
             );
           });
-
           cacheEncryptionKey &&
```

Comment:
Missing newline here as well:

```suggestion

          cacheEncryptionKey &&
```